index.html

```diff
@@ -188,15 +190,17 @@ <h4 class="languages1">Languages</h4>
       </ul>
     </div>
   </section>
+
+  <!-- Contact section -->
   <footer>
-    <section class="contact">
+    <section class="contact-box">
       <div class="cback">
         <h3 class="bold1">Contact me</h3>
         <p class="more">If you have an application you are interested in developing, a feature that you need built or a
           project that needs coding. I’d love to help with it </p>
         <br><br>
         <form>
-          <input type="text" maxlength="30" placeholder="Enter your name here" required>
+          <input type="text" maxlength="30" placeholder="Enter your Name" required>
```

Comment:
- [ ] Hello and nice job on your mobile app, however, you have missed some requirements, example: you should have used  [ formspree ](https://formspree.io/html/) to make your form submittable. please add this and resubmit for another review.